src/plugins/enzymes/src/FindEnzymesDialog.cpp

```diff
@@ -195,6 +222,10 @@ void EnzymesSelectorWidget::saveFile(const QString& url) {
     }
     if (QMessageBox::question(this, tr("New enzymes database has been saved."), tr("Do you want to work with new database?"), QMessageBox::Yes | QMessageBox::No) == QMessageBox::Yes) {
         loadFile(url);
+        if (!loadedEnzymes.isEmpty()) {
```

Comment:
These 5 lines of code are duplicated (from loadFile to emit a signal). Move them into a dedicated function.